RedHatInsights/ocp-advisor-frontend
# feat(OCPADVISOR-72): Render upgrade risks banner

Implements https://issues.redhat.com/browse/OCPADVISOR-72.
Mocked at https://www.sketch.com/s/c5c07e06-f4a1-4c64-9cdf-8b12d8ac9ad5/a/8ylYyDw.

This makes the cluster view render the upgrade risks banner which indicates whether there are any upgrade risks for this cluster.

- Must render the warning alert if there is at least one upgrade risks.
- Must render the success green alert if the are no upgrade risks for this cluster or the cluster is managed.
- Must render the warning alert "not currently available" if back end returns 503.
- Must not render the alert in any other cases. 

## How to test

The same as for https://github.com/RedHatInsights/ocp-advisor-frontend/pull/547#issue-1665942399: try to run the PR together with the mock service. For different clusters (scenarios), you should get different results.

## Screenshots

### Has some risks
![image](https://user-images.githubusercontent.com/31385370/232786763-c8906e84-bb62-48a9-8360-7adc6b5e9028.png)

### Has no risks/is a managed cluster
![image](https://user-images.githubusercontent.com/31385370/232786841-fe4e5ab4-ef05-4bc8-998b-cd8a8bc621af.png)

### Service is down (503 status code)
![image](https://user-images.githubusercontent.com/31385370/232786871-67b29bb8-d18f-46b9-b273-46566d46e011.png)
